src/ranch_conns_sup_sup.erl

```diff
@@ -21,7 +21,7 @@
 
 -spec start_link(ranch:ref(), module(), module(), module()) -> {ok, pid()}.
 start_link(Ref, Transport, Protocol, Logger) ->
-	ok = ranch_server:cleanup_connections_sups(Ref),
+	ok = ranch_server:cleanup_connections_sup_sup(Ref),
```

Comment:
Don't think there's a need to rename?

Reply:
Need, no, of course not :) I did that because now the function does _more_ than just throw aways the conns_sup pids, it now also throws away the counters ref. So it is more like "clean up connections sup sup related things".

Reply:
Yeah but let's keep the changes to a minimum.

Reply:
ok, i reverted it in my last commit